app/src/ui/lib/path-label.tsx

```diff
@@ -41,19 +44,17 @@ export class PathLabel extends React.Component<IPathLabelProps, {}> {
         ? availableWidth / 2 - ResizeArrowPadding
         : undefined
       return (
-        // eslint-disable-next-line jsx-a11y/label-has-associated-control
-        <label {...props}>
+        <span {...props} aria-hidden={this.props.ariaHidden}>
```

Comment:
Talked with Markus on this, I think the historical context was that the goal was to tie this label to the checkbox in the changes list. But, now that is applicable (plus it is used in other parts of the app that do not have check boxes like the history list). Therefore, this doesn't make sense as a span anymore.

Reply:
Note on aria-hidden, I found that even if I apply the `aria-label` attribute to the parent div that has the `role` of `option` as list items do. The default behavior is to use any child text as the accessibility label. Thus, this needs to be set to false in order for the `aria-label` to take precedence. 

Other things I tried was to apply an `aria-label` to the `svg` tag in the `octicon` component, but that was not picked up either. 

The only other thing that worked was to put a hidden div/span. That is still an option, but I think using the `aria-label` is programmatically more intentional/easier to understand what the purpose of at a glance.